### DISCUSSION TITLE
What is the fastest way to switch models? It's very slow or freezes.

### DISCUSSION THREAD

**Discussion by agentasylum** [vladmandic/sdnext#3645] [Q&A]
Hi. I'm not very experienced in using SD and webui, but I managed to properly install SDNext and a basic SDXL model. It generates images properly, but if I use the inpaint function and want to switch to an inpaint model, it takes 1-2 minutes for webui to switch to another model or it freezes completely and nothing happens. 

I saw that there are settings to make webui work with models faster, but I'm quite a beginner, I don't understand everything and I'm afraid I'll mess something up. Could you help me in detail with the best performance settings so that I can use SDNext comfortably? I have an NVidia GeForce RTX 460Ti video card with 16GB of memory. Thanks in advance!

**Comment by vladmandic** [comment]
switching models typically takes couple of seconds (3-4sec for sdxl on high end system up to 20-30sec on low-end), what are your models stored on? really need more info overal...

**Reply by agentasylum** [reply]
> switching models typically takes couple of seconds (3-4sec for sdxl on high end system up to 20-30sec on low-end), what are your models stored on? really need more info overal...

I use this models:
-stabilityai--stable-diffusion-xl-base-1.0
-thingthatis--stable-diffusion-xl-refiner-1.0

**Comment by vladmandic** [comment]
why use refiner? i mean, it is supported, but by now most of community gave up on it.
and you're saying you have issues switching, but what exactly do you mean - switching from what to what? you listed only one base model.

**Comment by agentasylum** [comment]
I tried switching to Juggernaut XL and back. I think my problem is that the development of the webui is very opaque, because you can't find anywhere whether the latest models now require a refiner or not, at least for beginners it's completely untraceable, questions like this and similar.

**Reply by vladmandic** [reply]
if you cannot find an answer online, you can always ask in discord community.
but regarding refiner, right now NO model requires it. never have. refiner has always been an optional thing. and with fine tuned models, its pretty much abandoned.

regarding slow swich, please show log that documents it and create issue as needed.
